javascript/javascript3/week1/main.js

```diff
@@ -0,0 +1,76 @@
+const ul = document.querySelector("#displayWeatherInfo");
+
+function unixToNormalTime(unixTimestamp) {
+  const date = new Date(unixTimestamp * 1000);
```

Comment:
This could be done much better with the `date.toLocaleTimeString('en-GB')` method. Check out the date methods you have available [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date).